components/fieldgroup/stories/fieldgroup.stories.js

```diff
@@ -62,6 +62,27 @@ export default {
 		inputType: "radio",
 		labelPosition: "top",
 		layout: "vertical",
+		label: "Select one of the following options:",
+		items: [
+			(passthroughs, context) => Radio({
+				...passthroughs,
+				id: "apple",
+				label: "Apples are best",
+				customClasses: ["spectrum-FieldGroup-item"],
+			}, context),
```

Comment:
We'll have to edit some of the `items` for certain stories. Right now, all of the checkbox variants are showing radio buttons instead of checkboxes.

<img width="448" alt="Screenshot 2024-09-26 at 9 14 19 AM" src="https://github.com/user-attachments/assets/9d5ebc2e-3eb2-4862-b002-9b762eac77c5">

Reply:
Also, does the toggle for `inputType` need to do anything else but change the `role` from `radiogroup` to `group`? It's doing that right now, all is good- I just expected it to change the component shown, but that seems like work for another PR.

Reply:
This is beyond the scope of this PR, but thanks for pointing out these regressions. I've added this to follow-up **CSS-969**.